koku/masu/database/sql/reporting_awscostentrylineitem_daily.sql

```diff
@@ -23,6 +23,7 @@ CREATE TEMPORARY TABLE reporting_awscostentrylineitem_daily_{{uuid | sqlsafe}} A
         sum(li.unblended_cost) as unblended_cost,
         max(li.blended_rate) as blended_rate,
         sum(li.blended_cost) as blended_cost,
+        sum(li.savingsplan_effective_cost) as savingsplan_effective_cost,
```

Comment:
Unless null values are OK, I suggest this:
```suggestion
        sum(coalesce(li.savingsplan_effective_cost, 0.0::numeric(33,15))) as savingsplan_effective_cost,
```

Reply:
updated 👍